src/algebra/homology/complex_shape.lean

```diff
@@ -133,20 +133,20 @@ end
 /--
 An option-valued arbitary choice of index `j` such that `rel i j`, if such exists.
```

Comment:
This comment isn't right any more, nore are any of the lemma names that mention `some`

Reply:
Fixed